File: javascript/1716.js
var totalMoney = function (n) {
    let output = 0
    let max_weeks = Math.ceil(n / 7)
    for (let i = 1; i < max_weeks + 1; i++) {
        for (let j = i; j < i + 7; j++) {
            output += j
            n -= 1
            if (n === 0) {
                break
            }
        }
    }
    return output
};

File: python/1716.py
import math


def totalMoney(n: int) -> int:
    output = 0
    max_weeks = math.ceil(n / 7)
    for i in range(1, max_weeks + 1):
        for j in range(i, i + 7):
            output += j
            n -= 1
            if n == 0:
                break
    return output
